packages/components/src/lib/theme.tsx

```diff
@@ -0,0 +1,39 @@
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+import type { PaletteRange } from '@mui/joy/styles';
+
+declare module '@mui/joy/styles' {
+  interface ColorPalettePropOverrides {
+    // apply to all Joy UI components that support `color` prop
+    brand: true;
+  }
+
+  interface Palette {
+    // this will make the node `secondary` configurable in `extendTheme`
+    // and add `secondary` to the theme's palette.
+    brand: {
+      blue: string;
+      lightBlue: string;
+    };
```

Comment:
Does this not have to be `brand: PaletteRange`? That's what the [docs](https://mui.com/joy-ui/customization/theme-colors/#adding-new-palettes) would suggest. Can you link in your PR description to whatever you're following? That's the only what i can judge correct-ness.

Reply:
Also this seems like it would expose `color="brand"` as an option for color-enabled components. But "brand" is not a valid color for us and is not our desired API. So this would be a wrong way to get these values in there.

Reply:
I tried `PalletteRange` and got the following error when I tried to set a value to `blue`:
```
Type '{ blue: string; lightBlue: string; }' is not assignable to type '{ 50?: string | undefined; 100?: string | undefined; 200?: string | undefined; 300?: string | undefined; 400?: string | undefined; 500?: string | undefined; 600?: string | undefined; 700?: string | undefined; ... 27 more ...; solidDisabledBg?: string | undefined; }
```
Which is why I went with being specific to each one.